src/pages/CONV_Account_Conversion.page

```diff
@@ -215,7 +215,7 @@
                             <svg aria-hidden="true" class="slds-button__icon slds-button__icon_large" data-dismiss="modal" data-target="warning_modal" viewBox="0 0 24 24">
                                 <path  data-dismiss="modal" data-target="warning_modal" d="M14.6 11.9l6-6c.3-.3.3-.7 0-1l-.9-1c-.3-.3-.7-.3-1 0L12.6 10c-.1.2-.4.2-.6 0L6 3.9c-.3-.3-.7-.3-1 0l-1 .9c-.3.3-.3.7 0 1l6.1 6.1c.1.1.1.4 0 .6L4 18.6c-.3.3-.3.7 0 1l.9.9c.3.3.7.3 1 0l6.1-6c.2-.2.5-.2.6 0l6.1 6c.3.3.7.3 1 0l.9-.9c.3-.3.3-.7 0-1l-6-6c-.2-.2-.2-.5 0-.7z"/>
                             </svg>
-                            <span class="slds-assistive-text">Close</span>
+                            <span class="slds-assistive-text">{!Label.bdiBtnClose}</span>
```

Comment:
It looks like this needs to be `$Label` to compile - though I'm not sure why MetaCI didn't pick this up

Reply:
Super weird, I saved it to my scratch org and verified it showed Closed in the inspector, I wonder if I actually got an error on save and didn't catch. TGIF, definitely having one of those days!